### ISSUE TITLE
Calling PluginContext.ParentContext gives error

### ISSUE THREAD

**Issue by harm-neervens-daf** [delegateas/XrmMockup#64]
When I try to get a reference to PluginContext.ParentContext, an exception I thrown: 'the gieven key is not present in the dictionary. Below the code as it looks after I rewrote it. Outcommented is the old code:
```csharp
public IPluginExecutionContext ParentContext
        {
            get
            {
                //return (IPluginExecutionContext)_propertyBag["ParentContext"];
                this._propertyBag.TryGetValue("ParentContext", out var result);

                return (IPluginExecutionContext)result;
                
            }
            set
            {
                _propertyBag["ParentContext"] = value;
            }
        }
```

**Comment by magesoe** [comment]
Well spotted! I was going to request a pull request, but realized how small the fix was, so I just did it. It has been fixed in version 0.9.5. It has just been published, so should be available in 5 minutes or so.

**Comment by magesoe** [comment]
Closed, assumed the fix worked
